ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -105,8 +106,9 @@ def is_filename_stdin(self, file_name):
 
     #---------------------------------------------------------------------
 
-    def is_filename_sairedis_rec(self, file_name):
-        return "sairedis.rec" in file_name
+    def require_marker_check(self, file_name):
+        files_to_skip = ["sairedis.rec", "bgpd.log"]
+        return not any([target in file_name for target in files_to_skip])
```

Comment:
Why not just do:
```Python
return  file_name not in files_to_skip
```

Reply:
Actually, `file_name` is the path to the file. If we want to check in this way, it needs to keep the full path of each file. I feel this is a little redundant. I correct `file_name` to `file_path` to avoid confusion.